### PR TITLE
Db setup

### PR DESCRIPTION
**Pull request by JustinD85**

### CODE REVIEW

**Comment by MillsProvosty** [comment]
Adds model specs and models for Users, Items, Orders and Order Items.

File: app/models/item.rb [hunk 1]
```diff
@@ -3,5 +3,6 @@ class Item < ApplicationRecord
   validates_numericality_of :price, :inventory
 
   belongs_to :user
-  
+  has_many :order_items
```
Comment (JustinD85) [inline app/models/item.rb:6]: :fire: :hankey:

Comment (MillsProvosty) [reply app/models/item.rb:6]: #classy

File: app/models/user.rb [hunk 2]
```diff
@@ -7,4 +7,5 @@ class User < ApplicationRecord
   validates_numericality_of :role
 
   has_many :items
+  has_many :orders
```
Comment (ryanmillergm) [inline app/models/user.rb:10]: These relationships look good!

**Review by davehardy632 — COMMENTED** [review]
We created database tables and models with the entire group.

**Review by ryanmillergm — APPROVED** [review]

File: app/models/item.rb (hunk 1 above)
Comment (ryanmillergm) [reply app/models/item.rb:6]: Looks good!

**Pull request merged**